src/Runtime.Base/src/System/Runtime/TypeCast.cs

```diff
@@ -42,6 +42,14 @@ internal enum AssignmentVariation
             AllowSizeEquivalence = 2,
         }
 
+#if CORERT
+        [RuntimeExport("RhTypeCast_IsInstanceOfClass_Generic")]
```

Comment:
The naming convention used for the helpers with RyuJIT convensions has been to just append "2" to the name. Also, I do not think this needs ifdef.